### PR TITLE
Heed nullability warnings in tools code

### PR DESCRIPTION
**Pull request by blairconrad**

### CODE REVIEW

File: tools/Directory.Build.props [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
     <Import Project="../Directory.Build.props" />
 
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>netcoreapp3.0</TargetFramework>
```
Comment (blairconrad) [inline tools/Directory.Build.props]: not strictly necessary, but I figured we'd get better annotations from the libraries, like knowing what `IsNullOrEmpty` means for its arguments.

File: tools/ReleaseHelpers.cs [hunk 2]
```diff
@@ -12,12 +13,10 @@ public static ICollection<int> GetIssueNumbersReferencedFromReleases(IEnumerable
...
             {
-                foreach (Match match in Regex.Matches(release.Body, @"\(\s*#(?<issueNumber>[0-9]+)(,\s*#(?<issueNumber>[0-9]+))*\s*\)"))
+                foreach (var capture in Regex.Matches(release.Body, @"\(\s*#(?<issueNumber>[0-9]+)(,\s*#(?<issueNumber>[0-9]+))*\s*\)")
+                                             .SelectMany(match => match.Groups["issueNumber"].Captures))
                 {
-                    foreach (Capture capture in match.Groups["issueNumber"].Captures)
-                    {
-                        issuesReferencedFromRelease.Add(int.Parse(capture.Value, NumberStyles.Integer, NumberFormatInfo.InvariantInfo));
-                    }
+                    issuesReferencedFromRelease.Add(int.Parse(capture.Value, NumberStyles.Integer, NumberFormatInfo.InvariantInfo));
```
Comment (blairconrad) [inline tools/ReleaseHelpers.cs]: There were warnings all in here, and I noticed that VS wanted to Linq up part of the query, so I let it, and... the warnings went away!

File: tools/Directory.Build.props (hunk 1 above)
Comment (thomaslevesque) [reply tools/Directory.Build.props]: :+1:

**Comment by blairconrad** [comment]
Thanks, @thomaslevesque. I figured we both needed a palate cleanser before what should be my last nullability PR: Specs. I'll start working on those tonight.

**Pull request merged**